funnel/templates/account_merge.html.jinja2

```diff
@@ -4,21 +4,21 @@
 
 {% macro accountinfo(user) %}
   <ul class="mui-list--aligned mui--text-subhead">
-    <li><strong>{% trans name=user.fullname %}Name:</strong> {{ name }}{% endtrans %}</li>
-    <li><strong>{% trans username=user.username or '' %}Username:</strong> {{ username }}{% endtrans %}</li>
+    <li><span class="mui--text-bold">{% trans name=user.fullname %}Name:{% endtrans %}</span> {{ name }}</li>
+    <li><span class="mui--text-bold">{% trans username=user.username or '' %}Username:{% endtrans %}</span> {{ username }}</li>
     <li>
       <ul>
         {%- for useremail in user.emails %}
-          <li><strong>{% trans email=useremail.email %}Email addresses:</strong>{{ email }}{% endtrans %}</li>
+          <li><span class="mui--text-bold">{% trans email=useremail.email %}Email addresses:{% endtrans %}</span>{{ email }}</li>
```

Comment:
Parameter not required. Space required between `</span>` and `{{ email }}`. Variable tag is incorrect as `email` is not a template variable. Change to `{{ useremail.email }}`